SensusService/Protocol.cs

```diff
@@ -387,6 +392,10 @@ public static void RunUnitTestingProtocol(Stream unitTestingProtocolFile)
         private bool _shareable;
         private List<PointOfInterest> _pointsOfInterest;
         private string _description;
+        private DateTime _startTime;
+        private bool _startImmediately;
+        private DateTime _endTime;
+        private bool _continueIndefinitely;
```

Comment:
Can we eliminate the two bools above, make the DateTimes nullable, and use null _startTime and _endTime to indicate indefinite?